DependencyInjection/Configuration.php

```diff
@@ -23,7 +23,7 @@ class Configuration implements ConfigurationInterface
      *
      * @return TreeBuilder
      */
-    public function getConfigTreeBuilder()
+    public function getConfigTreeBuilder(): TreeBuilder
```

Comment:
It is not necessary to drop support for PHP 5 here. You can instead use the special attribute to ommit the return type.

Reply:
`#[\ReturnTypeWillChange]`. it's php 8.1 only, but that's safe since php 8.2 is min version foe symfony 7.

Reply:
The type is require by the `ConfigurationInterface`

Reply:
Hmmm, that annotation only works with internal PHP interfaces. :/